docs/reference-guides/data/data-core-editor.md

```diff
@@ -555,6 +555,18 @@ _Returns_
 
 -   `string|undefined`: The post type label if available, otherwise undefined.
 
+### getPostTypeSupports
```

Comment:
Why is this a selector in the editor package?

Reply:
I have not thought about putting it here un purpose, it just came naturally after the `getPostTypeLabel` selector 🤷🏻

Reply:
I see, I was not aware of `getPostTypeLabel`. I agree that it's similar but it seems to me that neither should exist to be honest.

Reply:
Why tho? From the editor package readme:

> Having an awareness of the concept of a WordPress post, it associates the loading and saving mechanism of the value representing blocks to a post and its content. 

Since it knows of WordPress posts what is wrong with sugar selectors like these?

Reply:
I think we should avoid "convenient" selectors or shortcuts like that. There's more detailed reasoning here https://make.wordpress.org/core/2024/09/12/gutenberg-development-practices-and-common-pitfalls/ 

If we allow that, we'll never stop adding shortcuts and APIs everywhere. We just use core-data for these directly.